src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1018,7 +1018,9 @@ namespace nupic {
     for (UInt i = 0; i < numColumns; i++) {
       Real perm[8];
       sp.getPermanence(i, perm);
-      NTA_CHECK(check_vector_eq(truePermArr[i], perm, numInputs));
+      print_vec(truePermArr[i], 8);
+      print_vec(perm, 8);
+      NTA_CHECK(check_vector_eq(truePermArr[i], perm, numInputs)) << "Failing: " << truePermArr[i] << " vs. " << perm;
```

Comment:
What do I think about these lines specifically? I'd remove the `print_vec` statements but keep the error message with the `NTA_CHECK` but fix line length.